test/DebuggerCommon/step_in_from_interpreted_function_attach.js.dbg.baseline

```diff
@@ -2,19 +2,19 @@
   {
     "callStack": [
       {
-        "line": 27,
+        "line": 29,

```

Comment:
We do but only at top level under locals. Maybe we can revisit and fix ignoring at all levels.